<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainMenu</class>
 <widget class="QMainWindow" name="MainMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainMenu</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout_3" stretch="60,0,40">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="MenuLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 40px;
font-weight: bold;</string>
          </property>
          <property name="text">
           <string>French Number Game</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="ErrorLabel">
          <property name="styleSheet">
           <string notr="true">color:red;</string>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QGridLayout" name="gridLayout">
          <property name="leftMargin">
           <number>50</number>
          </property>
          <property name="topMargin">
           <number>50</number>
          </property>
          <property name="rightMargin">
           <number>50</number>
          </property>
          <property name="bottomMargin">
           <number>50</number>
          </property>
          <item row="1" column="1">
           <widget class="QLineEdit" name="HighestInput">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" alignment="Qt::AlignHCenter|Qt::AlignBottom">
           <widget class="QLabel" name="AmountLabel">
            <property name="text">
             <string>Amount</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0" alignment="Qt::AlignHCenter|Qt::AlignBottom">
           <widget class="QLabel" name="LowestLabel">
            <property name="text">
             <string>Lowest</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignBottom">
           <widget class="QLabel" name="HighestLabel">
            <property name="text">
             <string>Highest</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="3">
           <widget class="QPushButton" name="StartButton">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="LowestInput">
            <property name="text">
             <string>1</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="AmountInput">
            <property name="text">
             <string>10</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
